src/Infrastructure/Dto/Coordinates/Coordinates.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Dto\Coordinates;
+
+class Coordinates
+{
+    /**
+     * @var float
+     */
+    private $latitude;
```

Comment:
```suggestion
    private float $latitude;
```

we are using php7.4 already, so you can define typed properties and remove the annotation :)